types/webpack-env/index.d.ts

```diff
@@ -309,6 +309,11 @@ declare var __webpack_hash__: any;
  */
 declare var __non_webpack_require__: any;
 
+/**
+ * Initializes the shared scope. Fills it with known provided modules from this build and all remotes
+ */
+ declare var __webpack_init_sharing__: (scope: string) => Promise<void>;
```

Comment:
Is `__webpack_init_sharing__` always present or is it only defined in specific cases? Would it make sense to make this optional?

```suggestion
 declare var __webpack_init_sharing__: ((scope: string) => Promise<void>) | undefined;
```

Reply:
@rbuckton thanks for questions. 
__webpack_init_sharing__ will always present, just like __webpack_require__
https://github.com/webpack/webpack/blob/main/lib/APIPlugin.js#L24